src/wp-admin/plugin-install.php

```diff
@@ -24,6 +24,17 @@
 	exit;
 }
 
+if ( isset( $_GET['activate'] ) ) {
+	wp_admin_notice(
+		__( 'Plugin activated.' ),
+		array(
+			'id'                 => 'message',
+			'additional_classes' => array( 'updated' ),
+			'dismissible'        => true,
+		)
+	);
+}
```

Comment:
In case you're wondering why the `updated` class is added, this is what's used for [the notice](https://github.com/wordpress/wordpress-develop/blob/6.5/src/wp-admin/plugins.php#L723) on `plugins.php`, which uses [this array](https://github.com/wordpress/wordpress-develop/blob/6.5/src/wp-admin/plugins.php#L646). For consistency and to reduce the chances of unexpected differences, I used the same array data in `plugin-install.php`.